spec/formatting.md

```diff
@@ -235,6 +242,24 @@ rather than the _expression_ in the _selector_ or _pattern_.
 
 _Pattern selection_ is not supported for _fallback values_.
 
+### Expression Attribute Resolution
+
+The only _expression attribute_ that MUST be supported is `@locale`.
+If its value is set by a _literal_,
+it MUST be a comma-delimited sequence of IETF BCP 47 language tags.
+If its value is set by a _variable_,
+it MUST be either a single IETF BCP 47 language tag,
+or a sequence of IETF BCP 47 language tags.
```

Comment:
I wouldn't differentiate literals and variables, since the content ends up the same:

```suggestion
The value of `@locale` MUST be a comma-delimited sequence of well-formed IETF BCP47 language tags.
```

(we can provide instructions on processing the language priority list, including whitespace handling and such if needed)

Reply:
That works. Your proposed text already clarifies an empty value to be an error, as the sequence need to consist of "one or more" tags.